automatic/thunderbird/tools/chocolateyInstall.ps1

```diff
@@ -48,7 +52,7 @@ if (!(Get-32bitOnlyInstalled($softwareName)) -and (Get-OSArchitectureWidth 64))
 }
 
 Install-ChocolateyPackage @packageArgs
-if ($tbProcess) {
+if (!$pp.NoRestart -And $tbProcess) {
```

Comment:
You don't need this check. If an argument to not stop the process have been specified, the `$tbProcess` should have been set to null instead.